zebra-chain/src/sapling/shielded_data.rs

```diff
@@ -28,14 +28,18 @@ pub struct PerSpendAnchor {}
 #[derive(Copy, Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
 pub struct SharedAnchor {}
 
+/// This field is not present in this transaction version.
+#[derive(Copy, Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+pub struct FieldNotPresent {}
```

Comment:
I think this can literally be:

```suggestion
pub struct FieldNotPresent;
```

Reply:
Done!